hydra-main/src/main/java/com/addthis/hydra/job/spawn/balancer/SpawnBalancer.java

```diff
@@ -411,6 +413,32 @@ public boolean canReceiveNewTasks(HostState host) {
         return pushTasksOffHost(host, hosts, false, 1, moveLimit, false);
     }
 
+    private List<JobTaskMoveAssignment> getMoveAssignments(HostState host, Collection<HostState> otherHosts,
```

Comment:
Why doesn't this method take `moveLimit`? It could exit earlier if it did

Reply:
Since pushTasksOntoHost filters out some moveAssignments, this method returns a longer list than necessary. Would you recommend adding a moveLimit and moving fewer tasks onto light hosts (if they're filtered out)?